lib/features/sks_chart/data/repository/sks_chart_repository.dart

```diff
@@ -0,0 +1,24 @@
+import "package:fast_immutable_collections/fast_immutable_collections.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+import "package:riverpod_annotation/riverpod_annotation.dart";
+
+import "../../../../api_base_rest/client/dio_client.dart";
+import "../../../../config/env.dart";
+import "../models/sks_chart_data.dart";
+
+part "sks_chart_repository.g.dart";
+
+@riverpod
+Future<IList<SksChartData>> getLatestChartData(Ref ref) async {
```

Comment:
Did we always had the convention of `getX`? I thought we were using `XRepository` but everybody is doing `getSomething` lately

Reply:
Yeah, good catch. I changed it back to follow convention